packages/dm-core-plugins/src/tabs/Tabs.tsx

```diff
@@ -62,7 +65,9 @@ export const Tabs = (): JSX.Element => {
             onClick={() => setSelectedTab(tabData.attribute)}
             active={selectedTab === tabData.attribute}
           >
-            {tabData.entity.name}
+            {tabData.entity?.label ??
+              prettifyName(tabData.entity?.name || '') ??
+              selectedTab}
```

Comment:
shouldn't it be 
```suggestion
              tabData.entity?.attribute}
```
?

Reply:
It should!